pyanaconda/core/path.py

```diff
@@ -71,3 +71,40 @@ def mkdirChain(directory):
     """
 
     os.makedirs(directory, 0o755, exist_ok=True)
+
+
+def dir_tree_map(root, func, files=True, dirs=True):
+    """Apply a function to all paths under a directory.
+
+    Apply the given function to all files and directories in the directory tree
```

Comment:
Will the function be also applied on the root directory as well ? I think it would be good to specify that in this docstring to avoid later confusion.

Reply:
It uses `os.walk`, so no. Will add.